onnx/defs/schema.cc

```diff
@@ -502,7 +502,22 @@ OpSchema& OpSchema::AllowUncheckedAttributes() {
   allows_unchecked_attributes_ = true;
   return *this;
 }
-
+OpSchema& OpSchema::AddOpAnnotation(std::shared_ptr<OpAnnotation> annotation) {
+  this->op_annotations_.insert(*annotation);
+  this->op_annotation_flags_.insert(
+      annotation->GetAnnotations()->begin(),
+      annotation->GetAnnotations()->end());
+  return *this;
+}
+OpSchema& OpSchema::AddOpAnnotation(OpAnnotationFlag annotation_flag) {
+  auto registry = OpAnnotationRegistry::GetInstance();
+  return this->AddOpAnnotation(registry->GetOpAnnotation(annotation_flag));
```

Comment:
two return?

Reply:
What do you mean? What should we do in change in this function?

Reply:
The following instruction is also a `return`

Reply:
Oh sorry about that. I didn't see the `return *this` in the github view.